File: 20220907/모범답안/20220907_02.js/boat.py
def solution(People, Limit):
    People.sort()
    answer = 0

    while len(People) != 0:
        temp = People.pop()
        for i in range(len(People)-1, -1, -1):
            if Limit >= temp + People[i]:
                People.remove(People[i])
                break;
        answer += 1

    return answer

File: 20220907/모범답안/20220907_02.js/boat.js
function solution(people, limit)
{
  var answer = 0;
  var index = people.length - 1;

  people.sort(function(a, b)
  {
    return a - b;
  })

  while(people.length != 0)
  {

    var temp = people.pop();

    for(var i = people.length - 1; i >= 0; i--)
    {
      if(limit >= temp + people[i])
      {
        people.splice(i, 1);
        break;
      }
    }

    answer++;
  }

  return answer;
}